behave/features/5.e2e_not_eligible_postcode.feature

```diff
@@ -20,10 +20,4 @@ Feature: COVID-19 Shielded vulnerable people service - partial user journey - in
         Given I am on the "postcode-eligibility" page
         When I give the "#postcode" field the value "QJ5 7VC"
         And I submit the form
-        Then I am redirected to the "do-you-live-in-england" page
```

Comment:
i would revert this change as this is for when the tiering logic is set to false in the app